### ISSUE TITLE
Building on Windows 10

### ISSUE THREAD

**Issue by ghost** [UCI-UAVForge/GroundStation#98]
This may just be on my machine, but when I clean and rebuild the project I get an error in line 26 of ui_missionrecap.h.

To fix it, make the following change:

#include <qcustomplot.h> to #include "qcustomplot.h"

For a more permanent solution, is there a way to add qcustomplot.h to default libraries (like stdio.h, stdlib.h, iostream.h, etc.) or to have the uic write #include "qcustomplot.h" when compiling missionrecap.ui?

**Comment by JordanD2** [comment]
For the compiler error, I don't get this on my Mac, so I'd like to see exactly what's going wrong an where the error is on yours.

As for a long term solution, I have wanted to make qcustomplot a separate library for a while, so that we don't have to recompile it each time. I suspect that it is something like 30% of our compile/build time so fixing that would be great. I think that is one of the goals of issue #3

**Comment by ghost** [comment]
Yeah, I haven't been able to figure out how to do this in .pro files and such, but I could look into making QCustomPlot a dynamic library loaded at runtime. That way, we just recompile QCustomPlot when it's updated and our GS build won't have to deal with it.

**Comment by JordanD2** [comment]
@Roman-Parise, I think @tantiwup worked to reduce our compile times last Friday using the PRECOMPILE_HEADER tag in the .pro file. That might work until we get this packed away in some static library.

After doing some (very approximate) measurements, think that another major source of our long compile time is simply the GUI files. Quite a few of them get generated by the editor and they add up quick. I'm hoping that #124 and #132 help to clean this up a lot. I think we'll also be able to pack a lot of our other code (database manager, packet library, etc) into libraries as well if we really need to.
